### PR TITLE
allow separate author and destination repository

### PR DESCRIPTION
**Pull request by dmnemec**

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -5,13 +5,18 @@ FOLDER="$1"
 GITHUB_USERNAME="$2"
 GITHUB_REPO="$3"
 USER_EMAIL="$4"
+REPO_USERNAME="$5"
+
+if [ -z "$REPO_USERNAME" ] then
+  REPO_USERNAME= $GITHUB_USERNAME
```
Comment (cpina) [inline entrypoint.sh:11]: As far as I can tell: the space should not be there.

I think that it should be:
`REPO_USERNAME="$GITHUB_USERNAME"`

Do you care to change this? Then I can merge it.

And thanks for the PR!

Comment (dmnemec) [reply entrypoint.sh:11]: Hey there,

I can remove that space. I was also going to test this today to see if it actually solves the problem before requesting to merge it. Just didn't want to do it outside of business hours for us.

**Comment by dmnemec** [comment]
okay, got this working. Turns out it didn't work if the destination repo was set to use a different branch than master as the default.

**Comment by dmnemec** [comment]
Should probably test it still without the extra variable though, just to make sure the original behavior still works as expected.

**Comment by dmnemec** [comment]
I tested this without the new variable and it still works with it's original behavior. I think you're good to merge this whenever.

**Comment by cpina** [comment]
Thanks @dmnemec  - I forgot of merging it yesterday. I'll do it next!

**Comment by cpina** [comment]
Thanks very much for the other improvements as well (like --single-branch). I'm happy that you found this useful!

**Pull request merged**